### PR TITLE
Form artist crud

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]
Good for me !

**Review by LauraCourquin — APPROVED** [review]
good for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: backend/src/models/ArtistManager.js [hunk 1]
```diff
@@ -5,11 +5,9 @@ class ArtistManager extends AbstractManager {
 
   insert(artist) {
     return this.connection.query(
-      `insert into ${ArtistManager.table} (user_id,status,siren,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,cachet) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
+      `insert into ${ArtistManager.table} (user_id,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,galleryImg,  cachet, instagram, linkedin, status, siren) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
```
Comment (LoicBrassart) [inline backend/src/models/ArtistManager.js:8]: I may be wrong, but I think you have 
- 17 fields in your query
- 16 "?"
- 18 variables
Does this query behave as intended?

File: frontend/src/components/CardProfileInfo/index.jsx [hunk 2]
```diff
@@ -3,28 +3,67 @@ import linkedin from "@assets/imgProfile/linkedin.svg";
...
+      right: "auto",
+      bottom: "auto",
+      marginRight: "-50%",
+      transform: "translate(-50%, -50%)",
+      lineHeight: "2rem",
+      textAlign: "center",
+    },
+  };
+
+  const customStyles2 = {
```
Comment (LoicBrassart) [inline frontend/src/components/CardProfileInfo/index.jsx:66]: I had mised it the first time around, but that's crappy code

File: frontend/src/components/CardProfileInfo/index.jsx [hunk 3]
```diff
@@ -36,6 +75,7 @@ function CardProfileInfo() {
     },
   };
   const [modalIsOpen, setIsOpen] = useState(false);
+  const [modalIsOpen2, setIsOpen2] = useState(false);
```
Comment (LoicBrassart) [inline frontend/src/components/CardProfileInfo/index.jsx:78]: - Find better variables names
- I already read this code 1 PR ago -_-

**Review by LoicBrassart — APPROVED** [review]
Low quality (duplicated) code, merge at your own risk :shrug:

**Pull request merged**